docker/observation/.env.example

```
@@ -0,0 +1,32 @@
+GRAFANA_ADMIN_PASSWORD=admin
+
+# Prometheus Configuration
+PROMETHEUS_PORT=9090
+PROMETHEUS_RETENTION=7d
+
+# Tempo Configuration
+TEMPO_PORT=3200
+
+# Elasticsearch Configuration
+ELASTICSEARCH_PORT=9200
+ES_JAVA_OPTS=-Xms2g -Xmx2g
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)